manage-data/data-store/data-streams/failure-store.md

```diff
@@ -0,0 +1,2002 @@
+---
+applies_to:
+  stack: ga 8.19.0
+  serverless: ga 9.1.0
+---
+
+# Failure store [failure-store]
+
+Failure stores are a secondary set of indices inside a data stream dedicated to storing failed documents. Failed documents are any documents that cause ingest pipeline exceptions or have a structure that conflicts with a data stream's mappings. These failures normally cause the indexing operation to fail, returning the error message in the response. 
```

Comment:
```suggestion
A failure store is a secondary set of indices inside a data stream, dedicated to storing failed documents. A failed document is any document that, without the failure store enabled, would cause an ingest pipeline exception or that has a structure that conflicts with a data stream's mappings. In the absence of the failure store, a failed document would cause the indexing operation to fail, with an error message returned in the operation response.
```

I took a stab at rephrasing to incorporate Pete's comment above, and also to introduce the terms "failure store" and "failed document" as singular rather than plural, which is our usual convention, but it's just a suggestion. I think the original works nicely too.